### PR TITLE
CEDS-1412 Add test cases for MUCR association and dissociation

### PR DESCRIPTION
**Pull request by janeturner**

### CODE REVIEW

File: test/component/uk/gov/hmrc/exports/movements/AssociateUCRSpec.scala [hunk 1]
```diff
@@ -30,36 +30,72 @@ import uk.gov.hmrc.exports.movements.models.submissions.{ActionType, Submission}
...
+      }
+
+      "used for MUCR Association" in {
+        // Given
+        givenIleApiAcceptsTheSubmission("conversation-id")
+
+        // When
+        val response = post(
+          routes.ConsolidationController.submitConsolidation(),
+          Json.obj("providerId" -> "pid", "eori" -> "eori", "consolidationType" -> "ASSOCIATE_MUCR", "mucr" -> "MUCR", "ucr" -> "MUCR")
```
Comment (MaciejRewera) [inline test/component/uk/gov/hmrc/exports/movements/AssociateUCRSpec.scala]: I'd suggest to distinguish between two MUCRs. In this scenario you are associating the eMUCR with itself which is awkward.

Comment (MaciejRewera) [reply test/component/uk/gov/hmrc/exports/movements/AssociateUCRSpec.scala]: And what is even more important, you are not able to detect an error if you have switched those two in the process.

Comment (janeturner) [reply test/component/uk/gov/hmrc/exports/movements/AssociateUCRSpec.scala]: Good point. Could call the one to associate `associateMUCR`?

Comment (MaciejRewera) [reply test/component/uk/gov/hmrc/exports/movements/AssociateUCRSpec.scala]: I'd call it just `MUCR_2`. But it's up to you.

**Pull request merged**